2D Game/main.py

```diff
@@ -67,15 +63,22 @@
         total = elapsed_time
 
             
+    if keys[pygame.K_ESCAPE]:
+        try:
+            event.type = pygame.QUIT
```

Comment:
NOTE this code is incorrect in all cases (exception hanging does not allow you to edit immutable variables). Instead of seeing an error and crudely adding exception handling, investigate the error and attempt to eliminate it. This will improve your proficiency. I say this to be kind, senior devs would eat you up for this practice.

Reply:
Thanks for the comments. FYI, the "error" occurred after the program had already closed, so I didn't add too much thought to fixing it. I agree with your revised implementation.